UP_onboarding_system/point_of_sale/models.py

```diff
@@ -0,0 +1,56 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+
+class Profile(models.Model):
+    MERCHANT = 1
+    CONSUMER = 2
+
+    ROLE_CHOICES = (
+        (MERCHANT, 'MERCHANT'),
+        (CONSUMER, 'CONSUMER')
+    )
+
+    name = models.CharField(max_length=50)
```

Comment:
Also we already have `first_name` & `last_name` on User model, do we need additional field `name` on `Profile`

Reply:
I am using User model for merchant and consumer profile , so for merchants it easier to have 1 name , that's why I did it like that.